.travis.yml

```diff
@@ -22,6 +22,7 @@ before_script:
   - psql -c 'create database muesli;' -U postgres
   - psql -d muesli < travis-muesli-origDb.sql
   - cp muesli.yml.example muesli.yml
+  - export JWT_SECRET_TOKEN="secret"
```

Comment:
Well, until now we take the `JWT_SECRET_TOKEN` from the env.
It is a more Docker-conform approach to do so, but since the muesli is not running on docker we can ofc change this.